src/components/form/form.stories.js

```diff
@@ -0,0 +1,108 @@
+import { html } from "lit-html";
+import "../form";
+import "../form-grid";
+import "../input-field";
+import "../button";
+import "../form-message";
+import styles from "./styles.scss";
+import inputFieldStyles from "../input-field/styles.scss";
+import formGridStyles from "../form-grid/styles.scss";
+import buttonStyles from "../button/styles.scss";
+import formMessageStyle from "../form-message/styles.scss";
+import { stylesheet, docsIntro } from "../../../.storybook/utils.js";
+import { args, argTypes } from "./config";
+
+export default {
+  title: "native-elements/vl-form",
+  decorators: [
+    (story) =>
+      html`
+        ${stylesheet(
+        `${styles}${inputFieldStyles}${formGridStyles}${buttonStyles}${formMessageStyle}`
+      )}${story()}
+      `,
+  ],
+  parameters: {
+    docs: {
+      description: {
+        component: docsIntro({
+          stylesheets: ["form"],
+          root: "form",
+          intro: "Form element.",
+        }),
+      },
+    },
+  },
+  args,
+  argTypes,
+};
+
+export const Default = ({ validate }) => html`
+  <div style="max-width: 800px">
+    <p>
```

Comment:
Deze boodschap zou ik wel echt weglaten. De devs zien dat er een wrapper rond staat omdat we de wrapper in de story hebben gezet. Als we dit beginnen te doen moeten we consequent zijn en dat in elke story doen met een breedte-wrapper, en dat zijn er wel een paar.

Reply:
Fixed.